File: 입출력과 사칙연산/10단계-No.10430-나머지.c
#include <stdio.h>

int main(void) {
    int a, b, c, answer;
    
    scanf("%d %d %d", &a, &b, &c);
    
    answer = (a + b) % c;
    printf("%d\n", answer);
    printf("%d\n", answer%c);
    
    answer = (a * b) % c;
    printf("%d\n", answer);
    printf("%d\n", answer%c);

    return 0;
}

File: 입출력과 사칙연산/10단계-No.10430-나머지.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        
        int answer = (a+b)%c;
        System.out.printf("%d\n%d\n", answer, answer%c);
        
        answer = (a*b)%c;
        System.out.printf("%d\n%d", answer, answer%c);
        
        sc.close();
    }
}
